documentation/fiji_export/plugin.md

```diff
@@ -1,87 +1,49 @@
 ---
-title: FIJI import/export plugin
-tagline: Import and Export with a FIJI plugin
+title: ilastik FIJI plugin
+tagline: Import/export for ilastik results and running trained ilastik workflows in Fiji
 category: "Documentation"
 group: "basic-documentation"
 weight: 1
 ---
 
-# ilastik Import Export plugin in FIJI
-This plugin allows you to save images opened in FIJI directly to the hdf5 format preferred by ilastik. While this conversion is beneficial for any large dataset, we especially recommend it for _multipage tiffs_. After you save the file and process it with ilastik, you can use the same plugin to load the results for further post-processing in FIJI.
+# ilastik Fiji plugin
 
-## Installation
-In FIJI, go to _Help->Update..._ In the dialog that appears, click the "Manage update sites" button. In the list of update sites, select "ilastik". You should now see ilastik4ij.jar in your updater window.
-<div class="row">
-<div class="col-md-6">
-<a href="snapshots/update_sites.png" data-toggle="lightbox"><img src="snapshots/update_sites.png" width="100%" class="img-responsive" /></a>
-</div>
- <div class="col-md-6">
-<a href="snapshots/ImageJ_Updater.png" data-toggle="lightbox"><img src="snapshots/ImageJ_Updater.png" width="100%" class="img-responsive" /></a>
-</div>
-</div>
+This plugin allows you integrate ilastik with Fiji.
+The main functionality of the plugin is [import][docs-import]/[export][docs-export] to HDF5 format and running trained ilastik workflows from within Fiji.
```

Comment:
Change to "import and export"? Otherwise two hyperlinks kind of merge together visually.